tests/odbc/test-odbc-mssql.cpp

```diff
@@ -22,127 +22,613 @@ backend_factory const &backEnd = *soci::factory_odbc();
 // MS SQL-specific tests
 TEST_CASE("MS SQL long string", "[odbc][mssql][long]")
 {
-    soci::session sql(backEnd, connectString);
+  soci::session sql(backEnd, connectString);
 
-    struct long_text_table_creator : public table_creator_base
+  struct long_text_table_creator : public table_creator_base
+  {
+    explicit long_text_table_creator(soci::session &sql)
+        : table_creator_base(sql)
     {
-        explicit long_text_table_creator(soci::session& sql)
-            : table_creator_base(sql)
-        {
-            // Notice that 4000 is the maximal length of an nvarchar() column,
-            // at least when using FreeTDS ODBC driver.
-            sql << "create table soci_test ("
-                        "long_text nvarchar(max) null, "
-                        "fixed_text nvarchar(4000) null"
-                    ")";
-        }
-    } long_text_table_creator(sql);
-
-    // Build a string at least 8000 characters long to test that it survives
-    // the round trip unscathed.
-    std::ostringstream os;
-    for ( int n = 0; n < 1000; ++n )
+      // Notice that 4000 is the maximal length of an nvarchar() column,
+      // at least when using FreeTDS ODBC driver.
+      sql << "create table soci_test ("
+             "long_text nvarchar(max) null, "
+             "fixed_text nvarchar(4000) null"
+             ")";
+    }
+  } long_text_table_creator(sql);
+
+  // Build a string at least 8000 characters long to test that it survives
+  // the round trip unscathed.
+  std::ostringstream os;
+  for (int n = 0; n < 1000; ++n)
+  {
+    os << "Line #" << n << "\n";
+  }
+
+  std::string const str_in = os.str();
+  CHECK_NOTHROW((
+      sql << "insert into soci_test(long_text) values(:str)", use(str_in)));
```

Comment:
It would be nice to avoid mixing formatting changes like this with significant changes.

Besides, personally I liked the old format style better...